docs/3.api/4.commands/module.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "nuxi module"
+description: "Module utilities for your Nuxt application"
+links:
+  - label: Source
+    icon: i-simple-icons-github
+    to: https://github.com/nuxt/cli/blob/main/src/commands/module/add.ts
```

Comment:
```suggestion
    to: https://github.com/nuxt/cli/blob/main/src/commands/module/
```

Reply:
We might even remove the redundant trailing slash :)